src/Layout/Layout.scss

```diff
@@ -0,0 +1,15 @@
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+.h1,
+.h2,
+.h3,
+.h4,
+.h5,
+.h6,
+p {
+    text-align: start;
```

Comment:
I'm not sure we can safely add this property? As is, it breaks the headings on certain parts of the documentation site, for example (headings are no longer centered):

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/2828721/154689356-21ac962d-e223-4514-b2d1-a134186ed643.png">

Reply:
You are right. removed